invenio_theme/assets/semantic-ui/less/invenio_theme/theme/views/item.overrides

```diff
@@ -1,3 +1,23 @@
 /***********************************************
          Invenio Theme Item Overrides
 ***********************************************/
+
+.ui.dropdown .menu > .item.dropdown-item {
+    &:first-of-type:last-of-type{
+        padding: 1em 1em 1em 1em !important;
+    }
+
+    &:first-of-type{
+        padding: 1em 1em 5px 1em !important;
```

Comment:
why do we have a mixture of relative and not relative units here ? what problem does this CSS solve?

Reply:
This css was added when inside the mobile dropdown we had buttons instead of items. The sizes where the same than in another similar dropdown with buttons in the application. In any case, as we have Items now, it is no longer needed so I will delete it.